src/bp/sdk/botpress.d.ts

```diff
@@ -29,7 +29,7 @@ declare module 'botpress/sdk' {
     scope: string
     message: string
     metadata: any
-    timestamp: string
+    timestamp: any
```

Comment:
Typing should be Date. You can change the typing in logger.ts so it also works here, just need to make sure it's also outputted correctly with the file-persister